doc/apiv3-documentation.apib

```diff
@@ -552,17 +572,24 @@ Updates an activity's comment and, on success, returns the updated activity.
 
 # Group Attachments
 
-## Properties:
-| Property     | Description      | Type     | Constraints | Supported operations |
-| :---------:  | -------------    | ----     | ----------- | -------------------- |
-| id           | Attachment's id  | Integer  | x > 0       | READ                 |
-| fileName     |                  | String   |             | READ                 |
-| diskFileName |                  | String   |             | READ                 |
-| description  |                  | String   |             | READ                 |
-| contentType  |                  | String   |             | READ                 |
-| digest       |                  | String   |             | READ                 |
-| downloads    |                  | Integer  |             | READ                 |
-| createdAt    | Time of creation | DateTime |             | READ                 |
+## Linked Properties:
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This attachment                                     | Attachment    | not null    | READ                 |
+| container        | The object (e.g. WorkPackage) housing the attachment| Anything      | not null    | READ                 |
+| author           | The user that uploaded the attachment               | User          | not null    | READ                 |
+| downloadLocation | Direct download link to the attachment              | -             | not null    | READ                 |
```

Comment:
Didn't know that!

This sounds reasonable, let future us worry about the type issue!